src/example_fgen_basic/error_v/error_v.f90

```diff
@@ -41,14 +41,14 @@ module m_error_v
     end type ErrorV
 
     interface ErrorV
-    !! Constructor interface - see build (TODO: figure out cross-ref syntax) for details
+    !> Constructor interface - see docs [[m_error_v(module):build(subroutine)]] of for details.
```

Comment:
```suggestion
    !! Constructor interface - see docs [[m_error_v(module):build(subroutine)]] of for details.
```

docs not rendering here but not sure if this will just fix itself if we make the change above https://example-fgen-basic--32.org.readthedocs.build/en/32/fortran-api/interface/errorv.html